lib/OpenQA/WebAPI.pm

```diff
@@ -141,6 +142,13 @@ sub startup {
     # only provide a URL helper - this is overtaken by apache
     $r->get('/assets/*assetpath')->name('download_asset')->to('file#download_asset');
 
+    # placeholder types
+    my @links = [''];
```

Comment:
There should not be a need to initialize the variable if you just checking for a true value later on anyway.

Reply:
fixed